### PR TITLE
Update readme with multi gpu support and sample code

### LINKED ISSUE #227: Add documentation on how to use lightly on systems with multiple GPUs
Lightly supports multi-GPU setups using PyTorch Lightning using `ddp` in the trainer. 

- [ ] Add information about how to use lightly with multiple GPUs
- [ ] Add info to the readme that lightly supports multiple GPUs

### PR DESCRIPTION
**Pull request by IgorSusmelj**
### Changes

closes #227 

- add sample code on how to use lightly to readme
- add information about how to train lightly models on multiple gpus using pytorch lightning
- add information about how to use a custom PyTorch `Dataset`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,14 @@ Lightly is a computer vision framework for self-supervised learning.
 - [Github](https://github.com/lightly-ai/lightly)
 - [Discord](https://discord.gg/xvNJW94)
 
-### Supported Models
+### Features
+
+- modular framework
```
Comment (philippmwirth) [inline README.md:21]: Either add a sentence above, i.e. "Lightly offers features like" and then keep the small letters or change the first letter of every line to capital

File: README.md [hunk 2]
```diff
@@ -84,6 +61,83 @@ pip3 install lightly
 
 We strongly recommend that you install Lightly in a dedicated virtualenv, to avoid conflicting with your system packages.
 
+
+### Lightly in Action
+
+With lighlty you can use latest self-supervised learning methods in a modular
+way using the full power of PyTorch. Experiment with differnt backbones,
```
Comment (philippmwirth) [inline README.md:68]: differnt

File: README.md [hunk 3]
```diff
@@ -103,6 +157,35 @@ lightly-embed input_dir=/mydataset checkpoint=/mycheckpoint
 The embeddings with the corresponding filename are stored in a 
 [human-readable .csv file](https://docs.lightly.ai/getting_started/command_line_tool.html#create-embeddings-using-the-cli).
 
+
+### Benchmarks
+
+Currently implemented models and their accuracy on cifar10. All models have been evaluated using kNN. We report the max test accuracy over the epochs as well as the maximum GPU memory consumption. All models in this benchmark use the same augmentations as well as the same ResNet-18 backbone. Training precision is set to FP32 and SGD is used as an optimizer with cosineLR.
+One epoch on cifar10 takes ~35 secondson a V100 GPU. [Learn more about the cifar10 benchmark here](https://docs.lightly.ai/getting_started/benchmarks.html)
```
Comment (philippmwirth) [inline README.md:164]: secondson

File: README.md [hunk 4]
```diff
@@ -103,6 +157,35 @@ lightly-embed input_dir=/mydataset checkpoint=/mycheckpoint
 The embeddings with the corresponding filename are stored in a 
 [human-readable .csv file](https://docs.lightly.ai/getting_started/command_line_tool.html#create-embeddings-using-the-cli).
 
+
+### Benchmarks
+
+Currently implemented models and their accuracy on cifar10. All models have been evaluated using kNN. We report the max test accuracy over the epochs as well as the maximum GPU memory consumption. All models in this benchmark use the same augmentations as well as the same ResNet-18 backbone. Training precision is set to FP32 and SGD is used as an optimizer with cosineLR.
```
Comment (philippmwirth) [inline README.md:163]: we still need to benchmark barlow twins :)

**Review by philippmwirth — CHANGES_REQUESTED** [review]
More typos

File: README.md (hunk 2 above)
Comment (IgorSusmelj) [reply README.md:68]: Oh, you missed `lighlty`. Just saw this one now o.0

Comment (philippmwirth) [reply README.md]: maybe grep the docs for it?
```
grep -r lighlty docs/
```

**Review by philippmwirth — APPROVED** [review]

**Pull request merged**